packages/design-system/src/components/Promo/Promo.md

```diff
@@ -0,0 +1,58 @@
+### Promo
+
```

Comment:
Could you ask a UX designer for some general text about the component? As a new or external devcio I would not know what it's for and when to use it.

Reply:
```Promo banner are used to let users know about upgrade options, new features and any other informations that don't require immediate action. This component should be used wisely, don't over use it too often, and always specify action that you want to promote.```

Reply:
✅